Source/Resources/Content/MSGraph/DeviceManagement/GroupPolicyConfigurations/Security - LinkedIn Features in Office Applications Disabled.json

```diff
@@ -23,5 +23,5 @@
       "presentationValues": []
     }
   ],
-  "description": "Turned on in tenants by default, disable showing LinkedIn features in Office applications"
+  "description": "What is it: Turns off the Office LinkedIn features. \n Learn More: https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/linkedin-integration"
```

Comment:
**What does this do?**
The baseline turns off LinkedIN features in Office applications. By default, this setting is turned on.  

**Why should you use this?**
You can turn off Office LinkedIn features so that your users are not sharing data externally with LinkedIn. 

**What is the end user impact?**
Users will not be able to use the LinkedIn Office features. 

**Learn more**
[Integrate LinkedIn account connections in Azure Active Directory](https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/linkedin-integration)